test/harness.ts

```diff
@@ -1365,3 +1365,6 @@ export function waitForFileToExist(path: string, interval: number) {
     sleepSync(interval);
   }
 }
+
+// Ensure process.nextTick is available.
+process.nextTick(() => {});
```

Comment:
This is going to run on every test, which I don't think we want

Reply:
its always available we can just delete this change